### DISCUSSION TITLE
polling intervals

### DISCUSSION THREAD

**Discussion by DuraiDhyaan** [apache/skywalking#11875] [Q&A]
I am using Skywalking oap server and Skywalking java agent and Elastic search as storage.Each time when i go flow in my application i can able to see the traces. But after sometime the traces got disappeard by one by one . why cant i see the yesterday traces .Is there any limitation set for the traces to view. Once all the traces vanished .After restarting the application and OAP server , we can able to see the recent traces ..Also please help in setting the metrics
 polling time in skywalking/skywalking agent .

**Comment by wu-sheng** [comment]
This looks like the plugins you are using, not compatible in some specific cases.
So, when it happened, trace context leaked. You should be able to see kinds of warning in agent logs about this.

Generally, we don't have limitations for trace collecting. Many users run 100% sampling for billions of spans daily.

**Reply by DuraiDhyaan** [reply]
Thanks much for the quick response
